src/main/java/net/praqma/tracey/cli/Tracey.java

```diff
@@ -38,6 +34,7 @@ public static void main(String[] args) throws Exception {
         sayParser.addArgument("-n", "--node").dest("node").help("The URL of the RabbitMQ server");
         sayParser.addArgument("-e", "--exchange").setDefault("tracey").help("Exhange name");
         sayParser.addArgument("-u", "--user").dest("user").help("Username");
+        sayParser.addArgument("-h", "--headers").dest("headers").help("Add attribute file to the message");
         sayParser.addArgument("-s", "--secret").dest("secret").help("Password");
         sayParser.addArgument("-p", "--port").dest("port").type(Integer.class).help("Port");
         sayParser.addArgument("-c", "--configure").dest("config").help("Point to a config file");
```

Comment:
Also, mention that command line options will override values provided through the configuration file